cmd/root.go

```diff
@@ -128,6 +128,8 @@ func init() {
 	RootCmd.PersistentFlags().BoolVar(&options.SplitAnnualSummaryGains, "summarize-annual-gains", false,
 		"Summary will include transactions which represent annual capital gains/losses."+helpNl+
 			"Only valid with --summarize-before.")
+	RootCmd.PersistentFlags().BoolVar(&options.RenderTotalCosts, "total-costs", false,
+		"Print total costs across all securities")
```

Comment:
Related to other comment: We should just add a `(default, non-registered affiliate only)`

If someone has USD in their TFSA, they'll need to do some shenanigans with their csv to make this do what they want, for now.